### PR TITLE
Support door motion

### PR DESCRIPTION
**Pull request by dhylands**
I didn't bump the version, since I'll include these changes with the previous commits

### CODE REVIEW

File: zb-classifier.js [hunk 1]
```diff
@@ -83,22 +83,138 @@ const ZONE_TYPE_NAME = {
...
-  0x002c: {name: 'ped', label: 'Pressed', descr: 'Personal Emergency Device'},
+  0x0000: {
+    name: 'switch',
+    '@type': ['DoorSensor'],
+    propertyName: 'open',
+    propertyDescr: {
+      '@type': 'OpenProperty',
+      type: 'boolean',
+      label: 'Open',
+      descr: 'Contact Switch',
```
Comment (mrstegeman) [inline zb-classifier.js]: s/descr/description/g

File: zb-classifier.js [hunk 2]
```diff
@@ -83,22 +83,138 @@ const ZONE_TYPE_NAME = {
...
+    },
+  },
+  0x0028: {
+    name: 'fire',
+    '@type': ['BinarySensor'],
+    propertyName: 'on',
```
Comment (mrstegeman) [inline zb-classifier.js]: Most of these should not be named "on", otherwise they may end up conflicting.

File: zb-classifier.js [hunk 3]
```diff
@@ -83,22 +83,138 @@ const ZONE_TYPE_NAME = {
...
+      label: 'CO',
+      descr: 'Carbon Monoxide Sensor',
+    },
+  },
+  0x002c: {
+    name: 'ped',
+    '@type': ['BinarySensor'],
+    propertyName: 'on',
+    propertyDescr: {
+      '@type': 'BooleanProperty',
```
Comment (mrstegeman) [inline zb-classifier.js]: This could be `PushedProperty`

File: zb-classifier.js [hunk 4]
```diff
@@ -83,22 +83,138 @@ const ZONE_TYPE_NAME = {
...
+      label: 'Vibrating',
+      descr: 'Vibration/Movement Sensor',
+    },
+  },
+  0x010f: {
+    name: 'remote-panic',
+    '@type': ['BinarySensor'],
+    propertyName: 'on',
+    propertyDescr: {
+      '@type': 'BooleanProperty',
```
Comment (mrstegeman) [inline zb-classifier.js]: This could be `PushedProperty`

File: zb-classifier.js [hunk 5]
```diff
@@ -83,22 +83,138 @@ const ZONE_TYPE_NAME = {
...
+      label: 'Pressed',
+      descr: 'Remote Control',
+    },
+  },
+  0x0115: {
+    name: 'keyfob-panic',
+    '@type': ['BinarySensor'],
+    propertyName: 'on',
+    propertyDescr: {
+      '@type': 'BooleanProperty',
```
Comment (mrstegeman) [inline zb-classifier.js]: This could be `PushedProperty`

File: zb-classifier.js [hunk 6]
```diff
@@ -83,22 +83,138 @@ const ZONE_TYPE_NAME = {
...
+      label: 'Pressed',
+      descr: 'Keyfob',
+    },
+  },
+  0x021d: {
+    name: 'keypad-panic',
+    '@type': ['BinarySensor'],
+    propertyName: 'on',
+    propertyDescr: {
+      '@type': 'BooleanProperty',
```
Comment (mrstegeman) [inline zb-classifier.js]: This could be `PushedProperty`

File: zb-classifier.js (hunk 2 above)
Comment (dhylands) [reply zb-classifier.js]: There can only be one of these for a given sensor (since there is only one zoneType per sensor).

**Pull request merged**